main.py

```diff
@@ -0,0 +1,180 @@
+""" CS 180 - Organic Programming """
+from UserAccount import *
+
+def main():
+    exampleUser = UserAccount("kprada", "password")
+    accountList = [exampleUser]
+    menu(accountList)
+
+def menu(accountList):
+    choice = "P"
+    menuChoice = ""
+    loggedIn = False
+
+    while not loggedIn:
+        userAccountMenu()
+        choice = input("Choose an option: ")
+
+        # Login
+        if choice == "A":
+            account = login(accountList)
+            if not account:
+                print("Try Again?")
+            else:
+                loggedIn = True
+        # Register
+        elif choice == "B":
+            registerAccount(accountList)
+        # Reset Password
+        elif choice == "C":
+            print("Resetting password!")
+        else:
+            exit(0)
+
+    # Now we know they are logged in
+    while not menuChoice == "X":
+        landingPageMenu()
+        menuChoice = input("Choose an option: ")
+
+        # Create Database
+        if menuChoice == "A":
+            createMenu()
+        # Read Database
+        elif menuChoice == "B":
+            readMenu()
+        # Update Database
+        elif menuChoice == "C":
+            updateMenu()
+        # Delete Database
+        elif menuChoice == "D":
+            deleteMenu()
+        elif menuChoice == "E":
+            viewMenu()
+        else:
+            exit(0)
+
+
+def userAccountMenu():
+    print("[A] Login")
+    print("[B] Register")
+    print("[C] Reset Password")
+
+    print("[X] Quit")
+    print()
+
+def registerAccount(accountList):
+    print("REGISTER ACCOUNT")
+    username = input("Enter a username: ")
+    password = input("Enter a password: ")
+
+    user = UserAccount(username, password)
+    accountList.append(user)
+
+    print("Account created successfully!")
+    print()
+
+def login(accountList):
+    print("LOG-IN")
+    username = input("Username: ")
+    password = input("Password: ")
+
+    for i in range(len(accountList)):
+        if accountList[i].username == username and accountList[i].password == password:
+            print("Logged in successfully!")
+            print()
+            return accountList[i]
+        else:

```

Comment:
change to:
   for account in accountList:
        if account.getUsername() == username and account.getPassword() == password:
            print("Logged in successfully!")
            print()
            return account
    print("Invalid username or password.")
    return False

Reply:
check picture in #frontend